infer/src/IR/CallFlags.ml

```diff
@@ -17,17 +17,19 @@ type t =
   ; cf_injected_destructor: bool
   ; cf_interface: bool
   ; cf_is_objc_block: bool
+  ; cf_noreturn: bool
```

Comment:
> This was deleted by [9979489](https://github.com/facebook/infer/commit/997948914fec5af1954e399a8ca6ee91d41e2949) because it is dead code. It looks like the flag has been reintroduced by mistake in this PR.

Removed. Thanks